### DISCUSSION TITLE
What's the latest news about jQuery v4

### DISCUSSION THREAD

**Discussion by johnmark1215** [jquery/jquery#4990] [Q&A] [answered]
Hello,
For me, jQuery is amazing and really very powerful but there are some features that really need to be improved such as events to can support something like the 'passive' option.

So what's the expected date of releasing jQuery v4?

**Comment by mgol** [comment]
Hi,

thanks for your interest in jQuery 4. Unfortunately, there are a lot of factors like complexity of some tasks, availability of team members that changes over time, etc. so I cannot provide a planned release date. It'll be released when it's ready.

Until that happens, we'll continue to release minor or patch updates of the jQuery 3.x line.

**Reply by PaulDash** [reply]
jQuery 4 BETA 2 is here!

So... have passive listeners been added as an option? Can't find it in the Changelog. The issue was opened at https://github.com/jquery/jquery/issues/2871 but now moved to Milestone 5.0?

**Reply by mgol** [reply]
Yes, unfortunately, we had to postpone that until jQuery 5.0.0. We've already had a lot of changes done on the `main` branch and waiting for this & other originally planned issues could push it even further; we wanted to finally release a new major.

I'd like the time between jQuery 4.0.0 & 5.0.0 to be much shorter than between 3.0.0 & 4.0.0 but I cannot guarantee any timelines.

The best one can do is to help with fixing existing issues; then we can get to the final milestone much quicker.

**Comment by terabytesoftw** [comment]
Any news currently on JQUERY 4.0 or 5.0, will we have a final release?

**Reply by mgol** [reply]
We're currently preparing for `4.0.0-rc.1`. jQuery Migrate changes are almost ready, the upgrade guide is in progress. Once the upgrade guide is done, we'll release the RC.

**Comment by DrMuggi** [comment]
Any news about this 🙂?

**Reply by mgol** [reply]
We’re almost ready for the RC release; one small fix for the release process is required and then we only need some time for performing the release.

**Reply by objectboy2016** [reply]
> Any news about this 🙂?有关于这个🙂的消息吗？

Hope jQuery can regain its former glory.

**Reply by pyhoon** [reply]
It is going to be 1 year since the second beta of jQuery 4.0.0
https://blog.jquery.com/2024/07/17/second-beta-of-jquery-4-0-0/

**Comment by mgol** [comment]
The first release candidate for 4.0.0 is out now: https://blog.jquery.com/2025/08/11/jquery-4-0-0-release-candidate-1/

**Comment by mgol** [comment, marked as answer]
jQuery 4.0.0 stable is out: https://blog.jquery.com/2026/01/17/jquery-4-0-0/
